packages/common/src/routes.ts

```diff
@@ -0,0 +1,8 @@
+import { Address } from "wagmi";
+import { ChainId } from "./chains";
+
+export const Explorer = {
+  applicationPath(chainId: ChainId, roundId: Address, applicationId: string) {
```

Comment:
This is missing the return type, but that aside, is it worth making the parameters an object? The calling site likely already has variables with those exact names, so the call would only be two characters longer (braces), and would guard against this kind of compiler-undetectable bugs:

```tsx
  <a href={routes.applicationPath(roundId, chainId, applicationId)}> ... </a>
```

Reply:
you're right, fixed!